tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,312 @@
+- Set `trustCertificateKeyStorePassword` to your truststore password.
+
+</div>
+
+<div label="Python">
+
+[mysqlclient](https://pypi.org/project/mysqlclient/)'s TLS connection configurations are used here as an example.
+
+```
+host="tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com", user="root", password="<your_password>", port=4000, database="test", ssl_mode="VERIFY_IDENTITY", ssl={"ca": "ca.pem"}
+```
+
+You can click **show example usage** to view detailed code examples.
+
+```
+import MySQLdb
+
+connection = MySQLdb.connect(host="tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com", port=4000, user="root", password="<your_password>", database="test", ssl_mode="VERIFY_IDENTITY", ssl={"ca": "ca.pem"})
+
+with connection:
+    with connection.cursor() as cursor:
+        cursor.execute("SELECT DATABASE();")
+        m = cursor.fetchone()
+        print(m[0])
+```
+
+Parameter description：
+
+- Set `ssl_mode="VERIFY_IDENTITY"` to enable TLS and validate TiDB Dedicated Tier clusters.
+- Set `ssl={"ca": "<CA_root_path>"}` to set the CA root path on your system.
+
+</div>
+
+<div label="Go">
+
+[Go-MySQL-Driver](https://github.com/go-sql-driver/mysql)'s TLS connection configurations are used here as an example.
+
+```
+rootCertPool := x509.NewCertPool()
+pem, err := ioutil.ReadFile("ca.pem")
+if err != nil {
+    log.Fatal(err)
+}
+if ok := rootCertPool.AppendCertsFromPEM(pem); !ok {
+    log.Fatal("Failed to append PEM.")
+}
+mysql.RegisterTLSConfig("tidb", &tls.Config{
+    RootCAs:    rootCertPool,
+    MinVersion: tls.VersionTLS12,
+    ServerName: "tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com",
+})
+    
+db, err := sql.Open("mysql", "root:<your_password>@tcp(tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com:4000)/test?tls=tidb")
+```
+
+You can click **show example usage** to view detailed code examples.
+
+```
+package main
+import (
+  "crypto/tls"
+  "crypto/x509"
+  "database/sql"
+  "fmt"
+  "io/ioutil"
+  "log"
+  
+  "github.com/go-sql-driver/mysql"
+)    
+func main() {
+  rootCertPool := x509.NewCertPool()
+  pem, err := ioutil.ReadFile("ca.pem")
+  if err != nil {
+    log.Fatal(err)
+  }
+  if ok := rootCertPool.AppendCertsFromPEM(pem); !ok {
+    log.Fatal("Failed to append PEM.")
+  }
+  mysql.RegisterTLSConfig("tidb", &tls.Config{
+    RootCAs:    rootCertPool,
+    MinVersion: tls.VersionTLS12,
+    ServerName: "tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com",
+  })
+  db, err := sql.Open("mysql", "root:<your_password>@tcp(tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com:4000)/test?tls=tidb")
+  if err != nil {
+    log.Fatal("failed to connect database", err)
+  }
+  defer db.Close()
+
+  var dbName string
+  err = db.QueryRow("SELECT DATABASE();").Scan(&dbName)
+  if err != nil {
+    log.Fatal("failed to execute query", err)
+  }
+  fmt.Println(dbName)
+}
+```
+
+Parameter description：
+
+- Register `tls.Config` in connection to enable TLS and validate TiDB Dedicated Tier clusters. 
```

Comment:
What does "in connection" mean?  "in the connection string"?

Reply:
updated ，thanks